core/api/service/api_service.cpp

```diff
@@ -6,6 +6,25 @@
 #include "api/service/api_service.hpp"
 
 #include "api/jrpc/jrpc_processor.hpp"
+#include "api/jrpc/value_converter.hpp"
+
+namespace {
+  thread_local class {
+    boost::optional<kagome::api::Session::SessionId> binded_session_id_ =
```

Comment:
```suggestion
    boost::optional<kagome::api::Session::SessionId> bound_session_id_ =
```

Reply:
Why not just declare this variable globally, declared thread_local, by the way? Instead of wrapping it in a class which provides full get/set access to it.

Reply:
It's the only way to pass data between protocol layer and logic. So further data can be added here. ```class``` hides the data transfer method. ```declared thread_local, by the way``` exactly!

Reply:
Rename agree.

Reply:
Okay, if you plan to extend it then no problem.